style.css

```diff
@@ -119,4 +114,842 @@
     align-content: space-between;
     padding-right: 20px;
   }
+
+  .recent-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    width: 375px;
+    height: 3455px;
+    left: 0.01px;
+    top: 682px;
+  }
+
+  .frame44 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0.01px;
+    position: static;
+    width: 375px;
+    height: 3028px;
+    left: 0.01px;
+    top: 307px;
+    flex: none;
+    order: 1;
+    margin: 80px 0.01px;
+  }
+
+  .frame43 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0.01px 24px;
+    position: static;
+    width: 375px;
+    height: 2436px;
+    left: 0.01px;
+    order: 1;
+    margin: 37px 35.01px 24px 20px;
+  }
```

Comment:
- [ ] Kindly your `work` section overflows the page, and this's because of the margin and the fixed width, kindly fix it and review your all page because I see this issue with a lot of parts of your page.

![image](https://user-images.githubusercontent.com/35535130/159232087-496fc1b1-f029-4459-9a50-3b8ac3d36d6b.png)